docs/install.md

```diff
@@ -1,51 +1,73 @@
 # Installing Open Angel Arena
-You have to set up several things before you can run a local instance of AAA for development.
+You have to set up several things before you can run a local instance of OAA for development.
 
 ## System Dependencies
 Start by installing these programs:
  * [Steam](http://steampowered.com)
  * [nodejs](http://nodejs.org)
  * [Github client](http://desktop.github.com/)
- * [Sublime Text 3](http://www.sublimetext.com/) **optional**
+ * [Visual Studio Code](http://code.visualstudio.com/) `Text editor which includes support for github, editorconfig and much more`
```

Comment:
please remove this, VSC is an absolute garbage fire for dota modding purposes, we do not want to have to answer questions about it. Sublime text supports all of that shit, is much faster, is much more widely used, and we've already made it our standard.  Also this has nothing to do with gulp.